meta_package_manager/bitbar/meta_package_manager.7h.py

```diff
@@ -126,30 +151,36 @@ def print_menu():
         " ⚠️{}".format(total_errors) if total_errors else ""))
 
     # Print a full detailed section for each manager.
-    echo("---")
-    for manager in managers:
+    if recursive_menu:
+      submenu="--"
+    else:
+      submenu=""
 
+    if recursive_menu:
+        echo("---")
+    for manager in managers:
+        if not recursive_menu:
+            echo("---")
 
         if manager['error']:
-          # for line in manager['error'].strip().split("\n"):
-          #     echo("{:<23} {} | color=red font=Menlo size=10 trim=false emojize=false".format(manager.name + ':', line))
-          print_error(manager['error'])
+            print_error(manager['error'], manager['name'])
```

Comment:
What's your rational behind skipping the full sub-section of a manager when only one package is at fault?

`mpm`'s is currently returning a partial list of outdated packages even if one is triggering an error. That's the current implementation, and I like to keep it that way. Even if not complete, let's leverage this imperfect but legitimate list of outdated packages, on a best-effort basis.